examples/readme/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "tachijs-readme",
+  "version": "1.0.0",
+  "description": "readme example of tachijs",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "ts-node src/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BoostIO/tachijs.git"
+  },
+  "keywords": [
+    "readme",
+    "tachijs"
+  ],
+  "author": "Sheenan Tenepre",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/BoostIO/tachijs/issues"
+  },
+  "homepage": "https://github.com/BoostIO/tachijs#readme",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "class-validator": "^0.9.1",
+    "reflect-metadata": "^0.1.12",
+    "tachijs": "^0.4.1",
+    "tslib": "^1.9.3"
```

Comment:
Don't install tslib, but please install `typescript` and `ts-node` in `devDependencies`

Reply:
hmm for some reason, I've added `tslib` here. I don't think we need to add the `ts-node` and `typescript` package here since presumably, users should have installed these packages globally in their local. What do you think?